lntest/node.go

```diff
@@ -254,6 +254,13 @@ func (hn *HarnessNode) Name() string {
 	return hn.cfg.Name
 }
 
+// SetPort can be used to change P2P port of a node
+// TODO (Offer): remove once issue 1496 is resolved
+func (hn *HarnessNode) SetPort(port int)  {
```

Comment:
I think it would be best to wait until #1496 is resolved rather than adding this temporary hack.